### PR TITLE
Add Indian list

### PR DESCRIPTION
**Pull request by zloyden**
Let's take a look at https://github.com/AdguardTeam/FiltersRegistry/issues/608.
I think we can add it.
Language codes for India https://github.com/mediumkreation/IndianList#specialization -> https://developers.google.com/admin-sdk/directory/v1/languages

### CODE REVIEW

**Comment by DandelionSprout** [comment]
>can't find an indian tag

Probably because Indian isn't a language. You'd likely need to add the following tags instead: https://github.com/mediumkreation/IndianList#specialization

File: filters/ThirdParty/filter_253_IndianList/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "filterId": 253,
+  "name": "IndianList",
+  "description": "Additional filter list for websites in Indian.",
+  "timeAdded": 1650872701215,
+  "homepage": "https://github.com/mediumkreation/IndianList",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 7,
+  "subscriptionUrl": "https://easylist-downloads.adblockplus.org/indianlist.txt",
+  "tags": [
```
Comment (Alex-302) [inline filters/ThirdParty/filter_253_IndianList/metadata.json:11]: Not recommended?

Comment (zloyden) [reply filters/ThirdParty/filter_253_IndianList/metadata.json:11]: Actually, this is the first time I've seen this filter, that's why I didn't add `recommended` tag.
On the other hand, I don't see any alternatives.

File: tags/metadata.json [hunk 2]
```diff
@@ -262,5 +262,9 @@
   {
     "tagId": 66,
     "keyword": "lang:hr"
+  },
+  {
+    "tagId": 67,
+    "keyword": "lang:in"
```
Comment (slavaleleka) [inline tags/metadata.json:268]: not sure about `in`, seems like it's not valid language code for this filter and `hi` (for Hindi) and other should be added

**Review by slavaleleka — CHANGES_REQUESTED** [review]

File: tags/metadata.json (hunk 2 above)
Comment (Alex-302) [reply tags/metadata.json:268]: Yep
https://developers.google.com/admin-sdk/directory/v1/languages

Comment (zloyden) [reply tags/metadata.json]: Done.

**Review by Alex-302 — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Comment by DandelionSprout** [comment]
I feel that the `"description:" "Additional filter list for websites in Indian."` line is a bit odd.

Should probably be changed to `"Additional filterlist for websites in Indian and Sri Lankan languages."`

**Comment by zloyden** [comment]
> line is a bit odd.

Why is it? Their original [description](https://github.com/mediumkreation/IndianList#indianlist).
> This is an adblock filter list supplement for [EasyList](https://github.com/easylist/easylist) designed to block advertising on **Indian websites**.

**Comment by DandelionSprout** [comment]
Well, "Indian websites" means "Websites of India". Whereas "Websites in Indian" implies there's a singular Indian language.

Also, by some wild luck, the languages the list covers, also happen to cover Sri Lanka's two main languages (Sinhala and Tamil).

**Comment by slavaleleka** [comment]
filter description will be discussed and might be improved [later](https://github.com/AdguardTeam/FiltersRegistry/issues/610). let's merge it as it is

**Comment by aalpatkina** [comment]
We should write Hindi here

"Additional filter list for websites in Hindi"

**Pull request merged**